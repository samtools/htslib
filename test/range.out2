@HD	VN:1.4	SO:coordinate
@RG	ID:1	PL:ILLUMINA	PU:130410_HS18_09653_A_C1JT2ACXX_4	LB:7053878	DT:2013-04-10T00:00:00+0100	SM:ERS225193	CN:SC
@SQ	SN:CHROMOSOME_I	LN:1009800	M5:8ede36131e0dbf3417807e48f77f3ebd	UR:/
@SQ	SN:CHROMOSOME_II	LN:5000	M5:8e7993f7a93158587ee897d7287948ec	UR:/
@SQ	SN:CHROMOSOME_III	LN:5000	M5:3adcb065e1cf74fafdbba1e8c352b323	UR:/
@SQ	SN:CHROMOSOME_IV	LN:5000	M5:251af66a69ee589c9f3757340ec2de6f	UR:/
@SQ	SN:CHROMOSOME_V	LN:5000	M5:cf200a65fb754836dcc56b24b3170ee8	UR:/
@SQ	SN:CHROMOSOME_X	LN:5000	M5:6f9368fd2192c89c613718399d2d31fc	UR:/
@SQ	SN:CHROMOSOME_MtDNA	LN:5000	M5:cd05857ece6411f40257a565ccfe15bb	UR:/
@PG	ID:scramble	PN:scramble	VN:1.14.7	CL:scramble -M -I sam -s 50 -r /tmp/ce.fa - /tmp/ERR304769_subset.cram
HS18_09653:4:2108:14085:93656	147	CHROMOSOME_I	1122	60	100M	=	756	-466	AATTTGCAAGAAAATTCGCAAGAAATTTGTATTAAAAACTGTTCAAAATTTTTGGAAATTAGTTTAAAAATCTCACATTTTTTTTAGAAAAATTATTTTT	GEFGHHFHEGGIFEFHFH<HHGGEFIAHEEFGEHFHFDFGDHG@HGGFFIIHHG8HICFBCEGICHEGIBHEHH;CGGFDGGJFFHGGDGGFFFEGDDE?	X0:i:1	X1:i:0	XG:i:0	AM:i:37	SM:i:37	XM:i:0	XO:i:0	XT:A:U	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:2302:10109:87885	147	CHROMOSOME_II	1136	60	100M	=	734	-502	TGATTCATTTTATATTCTATATACTCATGTAATATGCCCATGTAAGGTTTAATTCCAAAAATATGAGCGTGTTCTATTTTATAATATTTTACTAAAATAC	GFGFEEGBEHH8BEFHCGGFEF.G:GB9FBFGEA@FB:<GF<G8FDFCGFCHBEHHDEEFDBGFCFHIEBCDE-ACGGGIFDDFFEFCHA=FDFAGDCC?	X0:i:1	X1:i:0	XG:i:0	AM:i:37	SM:i:37	XM:i:0	XO:i:0	XT:A:U	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:2108:10782:59721	163	CHROMOSOME_II	1241	60	100M	=	1366	225	AGTTAATTGCACTCAAATTTGTTGTTCTTCATTCTCTCGTTATGATTTAATCTTATTGCGTCAAGGTCATTATTTTAGGTCCATTAGTTATCGATCTGAA	?EDDGEFFFGGHIIHFGIHGHEHDHGDEHHGHGEHGHGHHGGHIGGGIGGIGIGGHHGHHGHHFFIHJGFHGHHGLGHGHGGHFKEHGIGGH@FHIFGFG	X0:i:1	X1:i:0	XG:i:0	AM:i:37	SM:i:37	XM:i:0	XO:i:0	XT:A:U	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:1107:10640:32305	99	CHROMOSOME_II	1267	60	100M	=	1778	611	CTTCATTCTCTCGTTATGATTTAATCTTATTGCGTCAAGGTCATTATTTTAGGTCCATTAGTTATCGATCTGAAACATGTTGTTGTATTTTTCTATTCTT	D?EFDEGGFFGEGHGGGHIGGGHHEGHIFGGEGFCGHFHGCHGGGHFKIHGGGGHFHFFHFFGGGGHGHHGHGFFHGGGHHGFGHGFGHFGGFHHBHGFF	X0:i:1	X1:i:0	BC:Z:GTNTGCNG	XG:i:0	AM:i:37	SM:i:37	XM:i:0	XO:i:0	QT:Z:<<!2@@!2	XT:A:U	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:2213:16876:56957	99	CHROMOSOME_II	1326	60	100M	=	1651	425	AGTTATCGATCTGAAACATGTTGTTGTATTTTTCTATTCTTGTGAGCTCAGGACACCTCATACAACTCCAGAGAAAATGTGTCTCATTATTCTTGTCTTT	BCEDDD:AAFGDG<F2DGHHGGGHDECGGG@GGFD/HGFGCB0GEEFGDHGHEGHFGHCFGHGFGHFHHHDFGFGHGEF.F-HGGBEGFGGGFHHGHEEI	X0:i:1	X1:i:0	BC:Z:GTNTGCNG	XG:i:0	AM:i:37	SM:i:37	XM:i:0	XO:i:0	QT:Z:<<!2@@!4	XT:A:U	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:2211:18838:86627	147	CHROMOSOME_II	1345	60	100M	=	828	-617	GTTGTTGTATTTTTCTATTCTTGTGAGCTCAGGACACCTCATACAACTCCAGAGAAAATGTGTCTCATTATTCTTGTCTTTTTTCAAGATCTAATCAATT	<GFDIEGH@FGDBFFEFFGFGAGHHAFH@HHGFHIHFGHGHG:CH:GCFHGGHGHGFEHGCJEJCGHGIGEEEFGFFGBHGHGEFHFCFGGG>FECDDE?	X0:i:1	X1:i:0	XG:i:0	AM:i:37	SM:i:37	XM:i:0	XO:i:0	XT:A:U	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:2314:21094:58611	99	CHROMOSOME_II	1353	60	100M	=	1775	522	ATTTTTCTATTCTTGTGAGCTCAGGACACCTCATACAACTCCAGAGAAAATGTGTCTCATTATTCTTGTCTTTTTTCAAGATCTAATCAATTTTCTACAT	D;?FBD9CDBGBGG?GF8DFGFFHDACDGFGGD/HGHHGFFEFGD=FGIG0D.GH7HHFFGFDGGFF:HFDGGHGGGGE;F:@GGEGGCFGFGHHB@FHG	X0:i:1	X1:i:0	BC:Z:GTNTGCCG	XG:i:0	AM:i:37	SM:i:37	XM:i:0	XO:i:0	QT:Z:=?!4AD22	XT:A:U	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:2108:10782:59721	83	CHROMOSOME_II	1366	60	100M	=	1241	-225	TGTGAGCTCAGGACACCTCATACAACTCCAGAGAAAATGTGTCTCATTATTCTTGTCTTTTTTCAAGATCTAATCAATTTTCTACATTAACGACGTTTTT	IFGHDHHFFGHIIEGGGHEHHJGGGFGGHFHHGFGGGGGGHHDHFHGIF=IFIFHIGIHGHF=HGJGGGFGGGHEEHGFGGFGEGGGGEGFFGGGFEBCD	X0:i:1	X1:i:0	BC:Z:GTNTGCCG	XG:i:0	AM:i:37	SM:i:37	XM:i:0	XO:i:0	QT:Z:=?!4AD+2	XT:A:U	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:2111:5602:28724	99	CHROMOSOME_II	1416	60	100M	=	1881	565	TCTTGTCTTTTTTCAAGATCTAATCAATTTTCTACATTAACGACGTTTTTGTCGTTCTGCTTCTTTTTTTCGTTCGTTTGTCTCGTCCATCAGCTGTCCA	ECE>EGGGGFGGGGDGFEFGGGFHEGHGIIFGFEJGHHFGGGHFGEFHIHGFFGGECGFHHGGFGHIHHHGEGGHBGBGHHEHGEBGGFFGFFHHGCGFF	X0:i:1	X1:i:0	BC:Z:GTNTGCCG	XG:i:0	AM:i:37	SM:i:37	XM:i:0	XO:i:0	QT:Z:=@!4AD24	XT:A:U	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:2103:6720:15025	99	CHROMOSOME_II	1459	60	100M	=	1617	258	CGTTTTTGTCGTTCTGCTTCTTTTTTTCGTTCGTTTGTCTCGTCCATCAGCTGTCCACTCATTTCTCTCCCACTCACTAGGCAGTGCTTTGTTTGGTTCC	ECEFFGGGEHGEGGGGGGHFGGGHIGHIGGGG?HFGHGEGFBFGGGFGIHGDGGDEDFF<FGGGGHFGGFFAGEGBGGCHFEFGGGEHEHGDGF:FFFFC	X0:i:1	X1:i:0	BC:Z:GTNTGCGG	XG:i:0	AM:i:37	SM:i:37	XM:i:0	XO:i:0	QT:Z:=?!4AD22	XT:A:U	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:1316:7415:40818	99	CHROMOSOME_II	1536	60	100M	=	1839	403	TAGGCAGTGCTTTGTTTGGTTCCGATTGGCAGCTGGCTGCAGGGCCTGCATCTCTTCTATGTCTCTCATTTACTTGCATTCTTTTCTTCGTTAATTTTTG	AC?>FGGAEFGGGDDGEGGFGGEEEGEIFGFG@E<GH>>EFGDG?HCFCF>DGGHDFFCHF>=G;CFBEHG<GCCGGEEHFDHGGHGGFFGGDDFHHGH?	X0:i:1	X1:i:0	BC:Z:GTNTGCGG	XG:i:0	AM:i:37	SM:i:37	XM:i:0	XO:i:0	QT:Z:=?!4AD+2	XT:A:U	MD:Z:100	NM:i:0	RG:Z:1
