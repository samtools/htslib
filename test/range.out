@HD	VN:1.4	SO:coordinate
@RG	ID:1	PL:ILLUMINA	PU:130410_HS18_09653_A_C1JT2ACXX_4	LB:7053878	DT:2013-04-10T00:00:00+0100	SM:ERS225193	CN:SC
@SQ	SN:CHROMOSOME_I	LN:1009800	M5:8ede36131e0dbf3417807e48f77f3ebd	UR:/
@SQ	SN:CHROMOSOME_II	LN:5000	M5:8e7993f7a93158587ee897d7287948ec	UR:/
@SQ	SN:CHROMOSOME_III	LN:5000	M5:3adcb065e1cf74fafdbba1e8c352b323	UR:/
@SQ	SN:CHROMOSOME_IV	LN:5000	M5:251af66a69ee589c9f3757340ec2de6f	UR:/
@SQ	SN:CHROMOSOME_V	LN:5000	M5:cf200a65fb754836dcc56b24b3170ee8	UR:/
@SQ	SN:CHROMOSOME_X	LN:5000	M5:6f9368fd2192c89c613718399d2d31fc	UR:/
@SQ	SN:CHROMOSOME_MtDNA	LN:5000	M5:cd05857ece6411f40257a565ccfe15bb	UR:/
@PG	ID:scramble	PN:scramble	VN:1.14.7	CL:scramble -M -I sam -s 50 -r /tmp/ce.fa - /tmp/ERR304769_subset.cram 
HS18_09653:4:2112:13048:11874	99	CHROMOSOME_II	2976	60	100M	=	3206	330	CTCAAGCTAATAGAGTATGGACAATTGTGAACGGAGAGGTTCAATGGAAGACTCCACCGCGGTAAGTGTGTTTCTTTAAAAATTACTTCCTTTTTTCAAT	DCEFDEGGFFGGGGGGFGGGGGGHDGHI?FGGGJGGHHGECGIH?HFGEHGHHGHEHHHHGHGGGGCGHHGHGHFGGGGHHGHGGGGIIGGGIGHHHG:G	X0:i:1	X1:i:0	BC:Z:GTGTGCGG	XG:i:0	AM:i:37	SM:i:37	XM:i:0	XO:i:0	QT:Z::=+4AD04	XT:A:U	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:1204:2530:59238	83	CHROMOSOME_IV	1422	0	100M	=	922	-600	ATCCGTTTGATGAGGAAGAGGAAGAAGAATCACAGTTTGGAGGGGGAACTCTGTCCGGTAGAGACCCATTTGATGAAGATGTACGTTTTTCGTAAAGTTC	GHCFGHFFFGGCCFHGGFG<HFGBHCEGC@FEGHGF=AFGFFFEFBG:HGHGGCHGFFDH>HEHGBJGGGGFGHFFH?@GHHEGGFAGE@BEGGFBDE?F	X0:i:2	X1:i:0	XA:Z:CHROMOSOME_IV,+46358,100M,0;	BC:Z:GTNTGCNG	XG:i:0	AM:i:0	SM:i:0	XM:i:0	XO:i:0	QT:Z:<<!2@@!4	XT:A:R	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:1303:14310:57578	83	CHROMOSOME_IV	1483	0	100M	=	988	-595	AGACCCATTTGATGAAGATGTACGTTTTTCGTAAAGTTCGGAATTTTTAGGGTATAAATTGAAAAACAAATGGAGAAAATATGAATACCTAGATTTTTTT	IBGHDCGFGGFIIEHGGFGHHFGGGGGGGDFHGFHGDEFDHFGHFGGHHGEIGFHFHFGIHEHHGGFGEGFGGHEIHEGGGFGGGDAGGGGGGGFDFECD	X0:i:2	X1:i:0	XA:Z:CHROMOSOME_IV,-1483,100M,0;	BC:Z:GTGTGCNG	XG:i:0	AM:i:0	SM:i:0	XM:i:0	XO:i:0	QT:Z:==+2@A!4	XT:A:R	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:2112:13048:11874	99	CHROMOSOME_II	2976	60	100M	=	3206	330	CTCAAGCTAATAGAGTATGGACAATTGTGAACGGAGAGGTTCAATGGAAGACTCCACCGCGGTAAGTGTGTTTCTTTAAAAATTACTTCCTTTTTTCAAT	DCEFDEGGFFGGGGGGFGGGGGGHDGHI?FGGGJGGHHGECGIH?HFGEHGHHGHEHHHHGHGGGGCGHHGHGHFGGGGHHGHGGGGIIGGGIGHHHG:G	X0:i:1	X1:i:0	BC:Z:GTGTGCGG	XG:i:0	AM:i:37	SM:i:37	XM:i:0	XO:i:0	QT:Z::=+4AD04	XT:A:U	MD:Z:100	NM:i:0	RG:Z:1
HS18_09653:4:1315:19857:61712	145	CHROMOSOME_I	914	23	78M1D22M	CHROMOSOME_V	1104758	0	AGCTAGGGCACTTTTTGTCTGCCCAAATATAGGCAACCAAAAATAATTTCCAAGTTTTTAATGATTTGTTGCATATTGAAAAAACATTTTTTGGGTTTTT	GGGFGHFDFG@IHG?DFGBFCGGHEEEHFFFGFCFFHIHGCHGHHIGIFHIGFHHIDH=GFADFGHHIHGEGIDGFFDGEGHFHFGFEEGFFFFEDDD<?	X0:i:1	X1:i:1	XA:Z:CHROMOSOME_X,-295167,78M1D22M,4;	XG:i:1	AM:i:0	SM:i:23	XM:i:2	XO:i:1	XT:A:U	MD:Z:48G29^A13C8	NM:i:3	RG:Z:1
HS18_09653:4:1308:11522:27107	161	CHROMOSOME_I	934	0	58M1D42M	CHROMOSOME_V	1104711	0	GCCCAAATATAGGCAACCAAAAATAATTTCCAAGTTTTTAATGATTTGTTGCATATTGAAAAAACATTTTTTGGGTTTTTTGAAATGAATATCATAGCTA	>CDDGEEFFFDHFFGDIJHEIJDGFFFIEIGIFFFFHGHFGF?BHHGGGGFGIGGHHGGFGHGFFIHGGHHEEHEDGHJHGGHBGGGFFGGEHHFHFGGG	X0:i:2	X1:i:11	XG:i:1	AM:i:0	SM:i:0	XM:i:3	XO:i:1	XT:A:R	MD:Z:28G29^A13C21G6	NM:i:4	RG:Z:1
HS18_09653:4:2314:14991:85680	83	CHROMOSOME_I	1020	10	28M72S	=	853	-195	TGAATATCATAGCTATAGAAACGGTAGTATTTTACTCTCTGTGGCTTCACAGTATATTTTACTCTCTGTGGCTTCACAGTATTTTTTACTCTCTGTGGCT	GGCFGHHGGGDHHGEGGHGH;FGGEGEGHFHHFBHBFGCJEHD<3BGH;GEGHFEGIFEIEFFHGG8GIEFFDHFIHFFGEHBFGDGGEGBFG5GEECCD	BC:Z:GTNTGCAG	XG:i:0	AM:i:10	SM:i:10	XM:i:3	XO:i:0	QT:Z:==!2@A+4	XT:A:M	MD:Z:8G6C9T2	NM:i:3	RG:Z:1
